ansible/roles/whiskhost/tasks/deploy.yml

```diff
@@ -2,6 +2,6 @@
 # Add user and patch kernel.
 
 - include: addUser.yml
-  when: "'prod' in deployment"
+  when: docker.user is defined
```

Comment:
Conventionally we use the following form in the **global** `group_vars/all`

```
docker
  user: foo
```

and this following form in **environment** `groups_var/all`

```
docker_user: foo
```

with the intention that vars in environments can be overridden from the CLI with `-e docker_user=bar`.

The global var could also be overridden (but preferably not), however it has be passed to the CLI as a *multi-hirarchy JSON format*   which is not human friendly.